dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
After a deeper analysis, it looks like we have several other tests that create unnamed Groups. Here's one: https://github.com/Georgetown-University-Libraries/DSpace/blob/ebf256caa1f03194cb547dae7eada64428a1b3e2/dspace-api/src/test/java/org/dspace/eperson/GroupTest.java#L125

So, the issue here seems to be that in a persistent DB (like Postgres), these unnamed groups persist.  We likely should be cleaning them up after their test runs/succeeds.